gtfs/gtfs_utils/gtfs_utils/gtfs_stats.py

```diff
@@ -93,6 +93,25 @@ def get_dates_to_analyze(use_data_from_today: bool, date_range: List[str]) -> Li
                 in range(delta.days + 1)]
 
 
+def validate_download_size(all_remote_keys: List[str], crud):
+    """
+    validate that the gtfs file to download are not too big
+    :param all_remote_keys: list of keys to be downloaded
+    :param crud: crud
+    :return: the total size of the file, IOError if the files are to big
+    """
+    files_size = get_files_size(all_remote_keys, crud)
+    free_space = get_free_space_bytes(configuration.files.full_paths.gtfs_feeds)
```

Comment:
I'd still leave the directory as a parameter, but with a default value of `configuration.files.full_paths.gtfs_feeds` as you used.
Also, are you sure that this file is the right location for this?

Anyway, this is my last comment for this review.